fighthealthinsurance/ml/health_status.py

```
@@ -304,5 +304,66 @@ def _refresh(self):
+    breakdown — ``{"name", "ok", "external", "error"}`` — for the staff-only
+    system status dashboard. It does not send alerts or mutate the cached
+    snapshot.
+
+    Checks run in parallel with a shared deadline; a backend whose check has
+    not finished by ``timeout_seconds`` is reported as not-ok with a timeout
+    error. Results are sorted problems-first (down before up, internal before
+    external, then by name) so on-call sees failures at the top.
+    """
+    try:
+        candidates = list(ml_router_module.ml_router.all_models_by_cost)
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)